site/js/runner.js

```diff
@@ -1,8 +1,8 @@
 var Runner = {
     editor: null,
     enabled: false,
-    inputString: '',
-    runStep: '',
+    inputString: 'abba',
```

Comment:
Why are we changing the default inputString here? Shouldn't this be set by the controller?